Jenkinsfile

```diff
@@ -333,29 +333,45 @@ def k8sTest(Map args = [:]) {
 */
 def withTools(Map args = [:], Closure body) {
   if (args.get('k8s', false)) {
-    withEnv(["KUBECONFIG=${env.WORKSPACE}/kubecfg"]){
-      retryWithSleep(retries: 2, seconds: 5, backoff: true){ sh(label: "Install kind", script: ".ci/scripts/install-kind.sh") }
-      retryWithSleep(retries: 2, seconds: 5, backoff: true){ sh(label: "Install kubectl", script: ".ci/scripts/install-kubectl.sh") }
-      try {
-        // Add some environmental resilience when setup does not work the very first time.
-        def i = 0
-        retryWithSleep(retries: 3, seconds: 5, backoff: true){
-          try {
-            sh(label: "Setup kind", script: ".ci/scripts/kind-setup.sh")
-          } catch(err) {
-            i++
-            sh(label: 'Delete cluster', script: 'kind delete cluster')
-            if (i > 2) {
-              error("Setup kind failed with error '${err.toString()}'")
-            }
+    withK8s() {
```

Comment:
🤔 if `k8s` is **false** then run with k8s. It is a little odd to me, the other way around makes more sense to me.

Reply:
that's not the case, it's the default value `map.get('value', otherwise-default-value)`

Reply:
I could have done:

```
def k8s = args.containsKey('k8s') ? args.k8s  : false
if (k8s)

```

but I used `if (args.get('k8s', false)) {` instead

Reply:
🤦‍♂️